### PR TITLE
Add name and label arguments to import

### PR DESCRIPTION
**Pull request by sbrunner**
I do this because when I want to import all my current documents calling all rename and add_label separately takes too many time...

### CODE REVIEW

File: paperwork-backend/paperwork_backend/shell.py [hunk 1]
```diff
@@ -520,22 +519,38 @@ def _do_import(filepaths, dsearch, doc, ocr=None, ocr_lang=None,
 
     for doc in import_result.new_docs:
         if guess_labels:
-            labels = dsearch.guess_labels(doc)
+            labels_ = dsearch.guess_labels(doc)
```
Comment (jflesch) [inline paperwork-backend/paperwork_backend/shell.py]: Please use a slightly more explicit name. Something like `guessed_labels`.

File: paperwork-backend/paperwork_backend/shell.py [hunk 2]
```diff
@@ -520,22 +519,38 @@ def _do_import(filepaths, dsearch, doc, ocr=None, ocr_lang=None,
...
-            for (label, scores) in labels:
+            for (label, scores) in labels_:
                 dsearch.add_label(doc, label, update_index=False)
+        for label_name, color in labels:
+            label = None
+            for clabel in dsearch.label_list:
+                if clabel.name == label_name:
+                    label = clabel
+                    break
+            if not label:
```
Comment (jflesch) [inline paperwork-backend/paperwork_backend/shell.py]: You can use `for ... else` here:

```
for clabel in dsearch.label_list:
    if clabel.name == label_name:
        label = clabel
        break
else:
    (...)
```
( http://book.pythontips.com/en/latest/for_-_else.html )

File: paperwork-backend/paperwork_backend/shell.py [hunk 3]
```diff
@@ -633,6 +653,17 @@ def cmd_import(*args):
         docid = args[idx + 1]
         args.pop(idx)
         args.pop(idx)
+    if "--name" in args:
```
Comment (jflesch) [inline paperwork-backend/paperwork_backend/shell.py]: Please update the documentation of the command: It's the docstring of the function `cmd_import`.

File: paperwork-backend/paperwork_backend/shell.py [hunk 4]
```diff
@@ -633,6 +653,17 @@ def cmd_import(*args):
...
+        idx = args.index("--name")
+        name = args[idx + 1]
+        args.pop(idx)
+        args.pop(idx)
+    while "--label" in args:
```
Comment (jflesch) [inline paperwork-backend/paperwork_backend/shell.py]: Idem: doc

**Review by jflesch — CHANGES_REQUESTED** [review]
First of all, thanks for contributing.

However, there are some small changes required (see other comments). Buildbot raised also a warning that must be fixed:
https://origami.openpaper.work/#/builders/2/builds/432
(you can run "make check" to check whether any other problem)

File: paperwork-backend/paperwork_backend/shell.py (hunk 2 above)
Comment (sbrunner) [reply paperwork-backend/paperwork_backend/shell.py]: Thanks, I don't know this, nice :-)

**Comment by jflesch** [comment]
Just one issue left regarding PEP8:
https://origami.openpaper.work/#/builders/2/builds/435

**Comment by sbrunner** [comment]
Hello @jflesch 
Fixed :-)
You run the jobs manually?
When I run `make check` I get different errors ...:
```
paperwork_backend/util.py:71:5: N802 function name should be lowercase
paperwork_backend/common/export.py:11:9: N805 first argument of a method should be named 'self'
paperwork_backend/img/page.py:159:9: N806 variable in function should be lowercase
paperwork_backend/img/page.py:160:9: N806 variable in function should be lowercase
```

**Comment by jflesch** [comment]
I start the buildbot manually yes. It's running flake8 on an debian stable, so it may not be exactly the same version you're running (`flake8 --version` == `3.2.1 (pyflakes: 1.3.0, pycodestyle: 2.2.0, mccabe: 0.5.3) CPython 3.5.3 on Linux`).
I've fixed the warnings you reported.

Your changes [are now OK](https://origami.openpaper.work/#/builders/2/builds/436), thanks :)

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**